libs/SalesforceAnalytics/build.gradle

```diff
@@ -52,3 +53,68 @@ afterEvaluate {
   // from timing out since executing tests are printed to the console.
   tasks.matching { it.name ==~ /^connected.*AndroidTest$/ }*.logging*.setLevel(LogLevel.INFO)
 }
+
+install {
+    repositories.mavenInstaller {
+        pom {
+            project {
+                packaging 'aar'
+                name 'SalesforceAnalytics'
+                url 'https://github.com/forcedotcom/SalesforceMobileSDK-Android'
+                licenses {
+                    license {
+                        name 'The Apache Software License, Version 2.0'
+                        url 'http://www.apache.org/licenses/LICENSE-2.0.txt'
+                    }
+                }
+                developers {
+                    developer {
+                        id 'bhariharan'
+                        name 'Bharath Hariharan'
+                        email 'bhariharan@salesforce.com'
+                    }
+                }
+                scm {
+                    connection 'https://github.com/forcedotcom/SalesforceMobileSDK-Android.git'
+                    developerConnection 'https://github.com/forcedotcom/SalesforceMobileSDK-Android.git'
+                    url 'https://github.com/forcedotcom/SalesforceMobileSDK-Android'
+
+                }
+            }
+        }
+    }
+}
+
+task sourcesJar(type: Jar) {
+    from android.sourceSets.main.java.srcDirs
+    classifier = 'sources'
+}
+
+artifacts {
+    archives sourcesJar
+}
+
+bintray {
+    user = System.getenv('BINTRAY_USER')
```

Comment:
This is setup such that it can be pulled from the env variables when you run the script. I have it setup on my `.bash_profile` for instance.